CRM/Mailing/BAO/SMSJob.php

```diff
@@ -46,12 +43,12 @@ public function deliver($unused, $isTest) {
       $mailing->subject = ts('[CiviMail Draft]') . ' ' . $mailing->subject;
     }
 
-    CRM_Mailing_BAO_Mailing::tokenReplace($mailing);
```

Comment:
I'm hoping this can go from the SMS emails entirely - but it is called twice now -here & in the mailing_preview api